VERSIONING.md

```diff
@@ -33,3 +33,35 @@ Compose version | Kotlin version
 1.0.0 | 1.5.31
 1.1.1 | 1.6.20
 1.2.0 | 1.7.10 - 1.7.20 (Desktop, Android), 1.7.10 (JS)
+
+### Relation to Jetpack Compose and Compose Multiplatform release cycle
+
+Compose Multiplatform shares a lot of code with [Jetpack Compose](https://developer.android.com/jetpack/compose) for Android developed by Google.
+We keep our release cycles aligned, so the common part is properly tested and stabilized.
+
+When the new minor version of Jetpack Compose is released, we pick the release commit, use it as a base for [Compose Multiplatform fork](https://github.com/JetBrains/androidx), finish new platform features, stabilize all platforms, and release Compose Multiplatform after some time.
+The gap between the Compose Multiplatform release and the Jetpack Compose release is usually 1-3 months.
+
+When you build your application for Android, the artifacts published by Google are used.
+
+In the next table you can see versions of the corresponding Jetpack Compose libraries.
+These are the versions on which we based Compose Multiplatform, and the versions used when you build your application for Android.
+
+Compose Multiplatform | Jetpack Compose Compiler | Jetpack Compose Runtime, UI, Foundation, Material | Jetpack Compose Material 3
+--- |--------------------|---------------------------------------------------| ---
+1.0.0 | 1.1.0-rc03 | 1.1.0-rc03                                        | 1.0.0-alpha03
+1.1.1 | 1.1.0 | 1.1.0                                             | 1.0.0-alpha05
+1.2.0 | 1.3.2 | 1.2.1                                             | 1.0.0-alpha14
+
+You may want to use [a cutting edge Compose Compiler](https://developer.android.com/jetpack/androidx/releases/compose-kotlin) published by Google for your Compose Multiplatform project.
```

Comment:
Consider adding: "The Compose Compiler version can be changed independently of other Compose libraries.  In order to support newer versions of Kotlin, " to the beginning of the sentence.

So this becomes:
The Compose Compiler version can be changed independently of other Compose libraries.  In order to support newer versions of Kotlin, you may want to use [a cutting edge Compose Compiler](https://developer.android.com/jetpack/androidx/releases/compose-kotlin) published by Google for your Compose Multiplatform project.

Reply:
Changed.